src/components/Note/NoteReactionList/NoteReactionBar/NoteReactionBar.scss

```diff
@@ -0,0 +1,60 @@
+@import "src/constants/style";
+
+.note-reaction-bar__root {
+  display: inline-flex;
+  flex-direction: row;
+  gap: $reaction__gap-size;
+
+  padding: $reaction__padding--outer-top-bottom $reaction__padding--outer-left-right $reaction__padding--outer-top-bottom 0; // left padding in NoteReactionList because of icon
+}
+
+.note-reaction-bar__reaction {
+  all: unset;
+
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+
+  width: $reaction__inner-size;
+  height: $reaction__inner-size;
+  padding: $reaction__padding--inner;
+
+  border-radius: $rounded--full;
+  font-size: $text--md;
+
+  cursor: pointer;
+
+  &:hover {
+    background-color: $hover-background-light;
+  }
+
+  &--active {
+    background-color: var(--accent-color--desaturated-light);
+
+    &:hover {
+      background-color: var(--accent-color--desaturated-light-hover);
+    }
+  }
+}
+
+@container #{$container__note} {
+  .note-reaction-bar__root {
+    gap: $reaction__gap-size--small;
+  }
+}
+
```

Comment:
I think the emoji buttons are too small on mobile. It'll be hard for users to press the right button. Maybe making the reaction bar full width on mobile and also increasing the height a bit might help

Reply:
<img width="373" alt="Bildschirmfoto 2023-09-28 um 11 45 12" src="https://github.com/inovex/scrumlr.io/assets/35272402/80400493-f851-4be8-82e0-657c3ab31510">

Reply:
I'll have to look into it if that's possible, because the emoji bar is actually already a tight fit, especially on our smallest supported device (iPhone SE, width=320px)

Reply:
update: this will be fixed later, see #3499